app/models/node.rb

```diff
@@ -390,8 +395,9 @@ def response_count(key = 'response')
   end
 
   # power tags have "key:value" format, and should be searched with a "key:*" wildcard
-  def has_power_tag(key)
-    !power_tag(key).blank?
+  def has_power_tag(key, preview_tags = [])
+    return !power_tag(key).blank? if preview_tags.blank?
+    match_preview_tag(key, preview_tags)
```

Comment:
Might this cause trouble, if another part of the app tries to use `has_power_tag()` in the old way? I just worry we're overloading this pretty simple function with some complicated and relatively limited-scope extra abilities that make it tough to understand and maintain. Let's see lower down...

Reply:
So here, could we change all instances of `preview_tags` to a more generalized name `override_tags` and add a comment above like `override_tags limits the tags within which to search to a predefined list`?

Reply:
Good idea, let me do that...